packages/features/form-builder/Components.tsx

```diff
@@ -392,7 +392,7 @@ export const Components: Record<FieldType, Component> = {
         if (!value) {
           setValue({
             value: options[0]?.value,
-            optionValue: "",
+            optionValue: options[0]?.label,
```

Comment:
`optionValue` is supposed to be added by the booker if applicable e.g. for Attendee Inputs. Also, it is a general purpose component that can be used by the booker to add more fields of this type.

Can we maybe tweak the options for locations so that there isn't a change in `radioInput` component and it receives the options[number].value as Label which would be practically unique.

Reply:
wait this is outdated

Reply:
can you pull again?